handlers/registration.js

```diff
@@ -28,25 +31,55 @@ module.exports.create = async (event, ctx, callback) => {
   }
   const id = parseInt(data.id, 10);
   const eventID = data.eventID;
+  let eventName = "";
   let registrationStatus = data.registrationStatus;
 
   // Check if the event is full
-  if (registrationStatus === 'registered'){
+  if (registrationStatus === 'registered') {
 
     const eventParams = {
       Key: { id: eventID },
       TableName: 'biztechEvents' + process.env.ENVIRONMENT
     }
-    
+
     await docClient.get(eventParams).promise()
-    .then(async(event) => {
```

Comment:
We still need this codeblock

Reply:
the code block is still there on my computer, maybe i should try pushing again?

Reply:
Sure